pages/mi/mi-10-steps.md

```diff
@@ -41,6 +41,9 @@ To participate in the internship, you will need the following hardware:
 
 Follow the instructions on [Creating Your GitHub Profile Page: A Guide to Markdown & Forking Workflow](mi-github-and-markdown.md).
 
+**Important: Only move onto the next step if you have created the pull request for your profile.**
+
+
```

Comment:
Could you remove this extra blank line for the sake of consistency across this file? Other than that it LGTM

Reply:
Fixed